test-e2e/tests/live_channel_test.ts

```diff
@@ -90,9 +90,6 @@ Scenario('I can watch the current live program on the live channel screen', asyn
   I.click('Start watching');
   I.seeElement('video');
 
-  // to make sure the back button is visible and can be clicked on
```

Comment:
It looks like the live channel on the demo app is currently broken. I've reached out to the live channels team to help fix it for us. I think we need to keep this line when it's working.

Reply:
The I.click call by itself makes sure the element is visible, no? If it isn't, the test is just going to fail, regardless of what's causing it. There were issues with the actual playback though yes, but I don't believe it makes a difference. The back button is also visible even when there's an error in playback (which could be a real use case) so it should be tested in those cases as well

Reply:
The first `click` is clicking on the video frame to make sure the controls are rendered (they fade out after a period of inactivity)